pkg/srv/mstream/event_builder.go

```diff
@@ -21,17 +21,19 @@ import (
 )
 
 type EventBuilder struct {
-	deviceName     string
-	defragmentedCh <-chan *layers.MStreamFragment
-	writerCh       chan<- []byte
+	deviceName         string
+	defragmentedCh     <-chan *layers.MStreamFragment
+	writerCh           chan<- []byte
+	api_headway_byteCh chan<- []byte
```

Comment:
please don't use underscores, use instead the camel notation

Reply:
I fixed it